modules/ztp-the-policygentemplate.adoc

```diff
@@ -168,3 +168,11 @@ spec:
                                     domainNumber 24
                                     .....
 ----
+
+The following example uses hub-side templating to generate multiple `machineConfigs` from a single {rh-rhacm} policy:
```

Comment:
The following example uses hub side templating to generate `PerformanceProfile` {rh-rhacm} policies for the cluster that have the following labels: `group-du-sno-zone: "zone-1"`, `hardware-type: "dell-poweredge-xr12"`.

Reply:
Thanks. I've proposed a change along these lines.